videointelligence/synth.py

```diff
@@ -66,6 +66,6 @@
 # Add templated files
 # ----------------------------------------------------------------------------
 templated_files = common.py_library(unit_cov_level=97, cov_level=100)
-s.move(templated_files, excludes="noxfile.py")
+s.move(templated_files)
```

Comment:
This part of #8666 probably needs to stay, in order not to have the next autogen undo #8607.